src/sql/workbench/services/objectExplorer/browser/serverTreeRenderer.ts

```diff
@@ -171,16 +171,14 @@ export class ServerTreeRenderer implements IRenderer {
 		}
 
 		let iconId = this._connectionManagementService.getConnectionIconId(connection.id);
-		if (!iconId) { return undefined; }
-
 		let providerProperties = this._connectionManagementService.getProviderProperties(connection.providerName);
 		if (!providerProperties) { return undefined; }
 
 		let iconPath: IconPath | undefined = undefined;
-		let pathConfig: URI | IconPath | { id: string, path: IconPath }[] | undefined = providerProperties.iconPath;
+		let pathConfig: URI | IconPath | { id: string, path: IconPath, default: boolean }[] | undefined = providerProperties.iconPath;
```

Comment:
default should be optional

(I would also suggest making this into a defined type so we don't have it duplicated between the two trees)